ios/browser/api/features/features.h

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_IOS_BROWSER_API_FEATURES_FEATURES_H_
+#define BRAVE_IOS_BROWSER_API_FEATURES_FEATURES_H_
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+OBJC_EXPORT
+@interface Feature : NSObject
+@property(nonatomic, readonly) NSString* name;
+@property(nonatomic, readonly) bool enabled;
+@end
+
+OBJC_EXPORT
+@interface FeatureList : NSObject
+@property(class, nonatomic, readonly) Feature* kARKillSwitch;
```

Comment:
can you use some macro for that? so this can look like:
```
OBJC_EXPORT
@interface FeatureList : NSObject
DECLARE_IOS_FEATURE(kARKillSwitch);
DECLARE_IOS_FEATURE(kAddressInfobarDisplayLength);
...
```

Reply:
https://github.com/brave/brave-core/pull/21559#discussion_r1460160024